### ISSUE TITLE
Change to TeX Management for Rstudio Side

### ISSUE THREAD

**Issue by d-alex-hughes** [berkeley-dsep-infra/hubs#4336]
### Package Name

tinytex

### Hub details

- Hub: ischool, but could generalize to stats/r based hubs
- Language: R 

Currently we're managing TeX installs through `tlmgr`. We found a working shortcut, using a frozen version (frozen to 2021) but this goes out of current annually in April. So far, we're one year/vintage behind (there is now a 2022 but we're written against 2021). 

One method of dealing with this is to use `TinyTex` managed by Yiuhi (the development lead for Rstudio/knitr/quarto). This way, they're maintaining the current install. 

PR #4335 proposes this change.

### Course Name

MIDS w203 is the primary user

**Comment by ryanlovett** [comment]
I should have commented here [instead of the PR](https://github.com/berkeley-dsep-infra/datahub/pull/4335#issuecomment-1458650923):
> @d-alex-hughes Fwiw, when we ran into the issue of students having to wait to install various TeX packages on other hubs, we [installed latex](https://github.com/berkeley-dsep-infra/datahub/blob/staging/deployments/stat20/image/Dockerfile#L29) from upstream Ubuntu packages. There haven't been any problems with it.

**Comment by d-alex-hughes** [comment]
Aha! I didn't want to propose something that far upstream -- but if it is working for Stat 20, and you want one fewer version of things, I'm glad to use that same setup. 

I can copy the Stat20 config over into my PR, yanking the `wget` for TinyTex; or, if there's a different way LMK and I'm happy to make the changes.

**Comment by d-alex-hughes** [comment]
I can also confirm, as a newly logged-in user on the Stat20 hub with no previous history, that it is functioning as I'd like it to for students.

**Comment by ryanlovett** [comment]
Okay, great. At one point we were appending lots of `tlmgr` invocations in stat20 but inevitably we'd miss something. Installing the ubuntu tex packages is more of a kitchen sink approach and make the image bigger, but I think it is worth it.

Inserting the following packages in the `apt-get` line in the ischool Dockerfile should be sufficient:
```
            texlive-xetex \
            texlive-latex-extra \
            texlive-fonts-recommended \
            texlive-lang-chinese \
            texlive-plain-generic \
```

**Comment by ryanlovett** [comment]
@d-alex-hughes I was able to merge your changes from https://github.com/berkeley-dsep-infra/datahub/pull/4335 in a separate PR -- CI was having problems with the other one for unknown reasons. Can you please test it at https://ischool-staging.datahub.berkeley.edu/ and then let us know if it can be merged to production?

**Comment by ryanlovett** [comment]
Fixed by #4355. @d-alex-hughes This is now in production.
